docs/_plugins/shortcodes/tokensTable.cjs

```diff
@@ -0,0 +1,11 @@
+module.exports = function(eleventyConfig) {
+  eleventyConfig.addPairedShortcode('tokensTable',
+    /**
+     * Tokens Table
+     * Display a table rows with token usage
+     */
+    function(content) {
```

Comment:
```suggestion
    function(content, options) {
```

include: string
exclude: string
only: 'tokens'|'cssProps'
types: string
element: string

I _think_ that would be enough? but if you think this is jumping the gun, let's defer until we talk it over with design

Reply:
Let's defer this till we know more what we want this shortcode to do.